.drone.yml

```diff
@@ -1,6 +1,3 @@
-env:
```

Comment:
Do you want to add `report-card: {}` to the publish block? This PR would remove it entirely.